src/main/scala/io/github/paoloboni/binance/spot/SpotApi.scala

```diff
@@ -192,6 +192,22 @@ final case class SpotApi[F[_]: Logger](
     } yield response
   }
 
+  def queryOrder(orderQuery: SpotOrderQueryParams): F[SpotOrderQueryResponse] =
```

Comment:
Could you please add a doc to this new API so that is becomes available here? https://paoloboni.github.io/binance-scala-client/latest/api/